include/gsl/gsl_assert

```diff
@@ -20,6 +20,13 @@
 #include <exception>
 #include <stdexcept> // for logic_error
 
+//
+// Temporary until MSVC STL supports no-exceptions mode.
+// Currently terminate is a no-op in this mode, so we add termination behavior back
+//
+#if defined(_MSC_VER) && !_HAS_EXCEPTIONS
```

Comment:
I think you want to test if _HAS_EXCEPTIONS is `#define`d before testing its value. People catch me making that mistake all the time.

Reply:
sure